package.json

```diff
@@ -37,8 +37,8 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json",
-    "test:ci": "jest --coverage --config ./test/jest-ci.json",
+    "test:e2e": "jest --coverage --config ./test/jest-e2e.json",
+    "test:all": "jest --coverage --config ./test/jest-all.json",
```

Comment:
My suggestion above applies here to. What do you think about having `test:all` and `test:all:cov`?

Reply:
[ac37ec1](https://github.com/5afe/safe-client-gateway-nest/pull/173/commits/ac37ec1366ece0316d011ad1e4c0ed4d860e705f)